extensions/mongo-client/deployment/src/main/java/io/quarkus/mongo/deployment/CodecProviderBuildItem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.mongo.deployment;
```

Comment:
I think I would have made the package `io.quarkus.mongodb.client` (same for the ITs).

But I can understand how it's not practical to change it now. So feel free to ignore.

Reply:
It needs to be done. I actually started, but forgot to commit and pushed. It will be done today.

Reply:
Done.